doc.go

```diff
@@ -24,5 +24,31 @@ After producing an output file, the assets may be accessed with the FS()
 function, which takes a flag to use local assets instead (for local
 development).
 
+HTTP Example
+
+Embedded assets can be served with HTTP using the http.FileServer. Assuming you have a directory called "static" and
+have run "esc static > static.go" and main.go has the following code:
+
+    package main
+
+    import (
+        "log"
+        "net/http"
+    )
+
+    func main() {
+        //FS() is created by esc and returns a http.Filesystem compatible with http.FileServer
+        http.Handle("/static/", http.FileServer(FS(false)))
+
+        //Start the server
+        if err := http.ListenAndServe(":8080", nil); err != nil {
```

Comment:
remove the comment on line 43, and just have: `log.Fatal(http.ListenAndServe(":8080", nil))` instead of this whole block